cmake/modules/RootConfiguration.cmake

```diff
@@ -493,31 +493,25 @@ if(veccore)
 else()
   set(hasveccore undef)
 endif()
-if(cxx11)
-  set(cxxversion c++11)
-  set(usec++11 define)
-else()
+
+set(cxxversion c++${CMAKE_CXX_STANDARD})
+
+if(CMAKE_CXX_STANDARD EQUAL 17)
```

Comment:
Can we `undef` all, and then `define` `usec++${CMAKE_CXX_STANDARD}`?

Reply:
Yes, good idea. I will do it like that.